spec/features/foods_index_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'food#index', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      id: 3,
+      name: 'foga',
+      email: 'kat@gmail.com',
+      password: 'test123',
+      confirmed_at: Time.now
+    )
+
+    sign_in @user1
+
+    @foods = [
+      Food.create(
+        id: 1,
+        user_id: @user1.id,
+        name: 'rice',
+        measurement_unit: 'kg',
+        quantity: 3,
+        price: 4
+      ),
+      Food.create(
+        id: 2,
+        user_id: @user1.id,
+        name: 'bread',
+        measurement_unit: 'gramms',
+        quantity: 4,
+        price: 1
+      )
+    ]
+
```

Comment:
- [ ] This test file is failing and so does many other files. Kindly work on the tests to pass as required. See the screenshot below of the failed tests 👍🏽

<img width="749" alt="Screenshot 2023-10-20 at 12 56 25 PM" src="https://github.com/tajulafreen/Recipe-app/assets/50941074/4e76e088-98fa-4df6-9862-f200da0cf05c">